commit 97645c4907c731ee8bb002ad7b36f89a0180e058
Author: Neil Horman <nhorman@tuxdriver.com>
Date:   Tue Jun 17 16:00:27 2014 -0400

    tmon: set umask to a reasonable value
    
    current, the tmon umask value is set to 0, which mean whatever the permission
    mask in the shell are when starting tmon in daemon mode are what the permissions
    of any created files will be.  We should likely set something more explicit, so
    lets go with the usual 022
    
    Signed-off-by: Neil Horman <nhorman@tuxdriver.com>
    CC: Zhang Rui <rui.zhang@intel.com>
    CC: Jacob Pan <jacob.jun.pan@linux.intel.com>

diff --git a/tools/thermal/tmon/tmon.c b/tools/thermal/tmon/tmon.c
index 059e0be..09b7c32 100644
--- a/tools/thermal/tmon/tmon.c
+++ b/tools/thermal/tmon/tmon.c
@@ -355,7 +355,7 @@ static void start_daemon_mode()
 	disable_tui();
 
 	/* change the file mode mask */
-	umask(0);
+	umask(S_IWGRP | S_IWOTH);
 
 	/* new SID for the daemon process */
 	sid = setsid();
